### PR TITLE
TAJO-1860: Refactor Rpc clients to take Connection Parameters.

### PR DESCRIPTION
**Pull request by hyunsik**

### CODE REVIEW

File: tajo-core-tests/src/test/java/org/apache/tajo/util/TestRpcConnectionParamUtil.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package org.apache.tajo.util;
```
Comment (jinossy) [inline tajo-core-tests/src/test/java/org/apache/tajo/util/TestRpcConnectionParamUtil.java]: you should add license

File: tajo-rpc/tajo-rpc-protobuf/src/main/java/org/apache/tajo/rpc/AsyncRpcClient.java [hunk 2]
```diff
@@ -18,56 +18,53 @@
...
-    init(new ProtoClientChannelInitializer(handler,
-        RpcResponse.getDefaultInstance(),
-        timeUnit.toNanos(timeout),
-        enablePing), eventLoopGroup);
+
+    final long socketTimeoutMills = Long.parseLong(
+        connectionParameters.getProperty(CLIENT_SOCKET_TIMEOUT, String.valueOf(CLIENT_SOCKET_TIMEOUT_DEFAULT)));
+
+    init(new ProtoClientChannelInitializer(handler, RpcResponse.getDefaultInstance(),
+        TimeUnit.MILLISECONDS.toNanos(socketTimeoutMills)),eventLoopGroup);
```
Comment (jinossy) [inline tajo-rpc/tajo-rpc-protobuf/src/main/java/org/apache/tajo/rpc/AsyncRpcClient.java]: You should remove toNanos()

File: tajo-rpc/tajo-rpc-protobuf/src/main/java/org/apache/tajo/rpc/BlockingRpcClient.java [hunk 3]
```diff
@@ -18,56 +18,54 @@
...
-    init(new ProtoClientChannelInitializer(handler,
-        RpcResponse.getDefaultInstance(),
-        timeUnit.toNanos(timeout),
-        enablePing), eventLoopGroup);
+
+    long socketTimeoutMills = Long.parseLong(
+        connectionParameters.getProperty(CLIENT_SOCKET_TIMEOUT, String.valueOf(CLIENT_SOCKET_TIMEOUT_DEFAULT)));
+
+    init(new ProtoClientChannelInitializer(handler, RpcResponse.getDefaultInstance(),
+        TimeUnit.MILLISECONDS.toNanos(socketTimeoutMills)),eventLoopGroup);
```
Comment (jinossy) [inline tajo-rpc/tajo-rpc-protobuf/src/main/java/org/apache/tajo/rpc/BlockingRpcClient.java]: You should remove toNanos()

File: tajo-rpc/tajo-rpc-protobuf/src/main/java/org/apache/tajo/rpc/AsyncRpcClient.java (hunk 2 above)
Comment (hyunsik) [inline tajo-rpc/tajo-rpc-protobuf/src/main/java/org/apache/tajo/rpc/AsyncRpcClient.java]: I missed them after the change ProtoClientChannelInitializer.

**Comment by hyunsik** [comment]
I've updated the jdbc documentation.
http://people.apache.org/~hyunsik/TAJO-1860/jdbc_driver.html

**Comment by hyunsik** [comment]
I fixed bugs and reflected your comments.

**Comment by jinossy** [comment]
+1 Looks good to me

**Pull request closed without merging**